tests/npbench/polybench/simple_add_test.py

```diff
@@ -0,0 +1,100 @@
+## copied jacobi 1d and simplified the kernel
```

Comment:
I would remove change a bit this test:
- you can move it under the `tests/fpga` folder, and give a name more related to the thing that wants to test (something like "long_long_opencl_test.py")
- I would add a brief comment on top explaining what's the purpose of the test
- I would test only FPGA, and in particular intel_fpga. So no CPU/GPU test.  To run only for Intel FPGA you can mark the test with `@intel_fpga_test()` rather than `@fpga_test`
- Cleanup the code, removing CPU/GPU related code
- no need to have all the sizes, just use and pick one
- remove unused imports